chatcommands.py

```diff
@@ -2043,6 +2043,12 @@ def delete(msg):
                "`sd delete-force`. See [this note on message deletion]"\
                "(https://charcoal-se.org/smokey/Commands"\
                "#a-note-on-message-deletion) for more details."
+    elif msg.room.id == 11540:
+        return "Messages here are usually kept as records."\
+               "If you really need to delete a message, please use "\
+               "`sd delete-force`. See [this note on message deletion]"\
+               "(https://charcoal-se.org/smokey/Commands"\
+               "#a-note-on-message-deletion) for more details."
```

Comment:
Instead of adding a whole new block here, simply remove the check for `post_data` on line (currently) 2040, so that it becomes simply `if msg.room.id == 11540:`

Reply:
No, because the 2 messages are different, unless you want them as one?